include/config/config_debug.h

```diff
@@ -94,3 +94,9 @@
  * Intentionally crash the game whenever a runtime assertion fails (also invoked by the DEBUG define in the Makefile).
  */
 #define DEBUG_ASSERTIONS
+
+/**
+ * Crash Screen: enable symbol-assisted stack trace
+ *  If disabled, stack trace will just dump memory, and you can figure it out yourself.
+ */
+#define DEBUG_FULL_STACK_TRACE
```

Comment:
Nit: This should probably either be included as part of the `DISABLE_ALL` safeguard, or come with a disclaimer that it isn't affected by it. Assuming this has no real impact on releases, I'd advocate for the latter.

Reply:
addressed